tests/foreman/cli/test_repository.py

```diff
@@ -2385,17 +2385,24 @@ class TestAnsibleCollectionRepository:
     @pytest.mark.upgrade
     @pytest.mark.parametrize(
         'repo_options',
-        **parametrized(
-            [
-                {
-                    'content-type': 'ansible_collection',
-                    'url': ANSIBLE_GALAXY,
-                    'ansible-collection-requirements': '{collections: [ \
+        [
+            {
+                'content-type': 'ansible_collection',
+                'url': ANSIBLE_GALAXY,
+                'ansible-collection-requirements': '{collections: [ \
                             { name: theforeman.foreman, version: "2.1.0" }, \
                             { name: theforeman.operations, version: "0.1.0"} ]}',
-                }
-            ]
-        ),
+            },
+            {
+                'content-type': 'ansible_collection',
+                'url': settings.ansible_hub.url,
+                'ansible-collection-auth-token': settings.ansible_hub.token,
+                'ansible-collection-auth-url': settings.ansible_hub.sso_url,
+                'ansible-collection-requirements': '{collections: \
+                                                         [redhat.satellite_operations ]}',
+            },
+        ],
+        ids=['ansible_gallaxy', 'ansible_hub'],
```

Comment:
:+1: Great call setting ids here instead of just using `**parametrized` to give them integer indices.

Reply:
One `l` in `ansible_galaxy` though